test/files/run/compiler-asSeenFrom.scala

```diff
@@ -148,10 +102,13 @@ package ll {
 
   def pretty(xs: List[_]) = if (xs.isEmpty) "" else xs.mkString("\n  ", "\n  ", "\n")
 
-  def signaturesIn(info: Type): List[String] = (
-    info.members.toList
-      filterNot (s => s.isType || s.owner == ObjectClass || s.owner == AnyClass || s.isConstructor)
-      map (_.defString)
+  def signaturesIn(sym: Symbol): List[String] = (
+    if (sym.owner == ObjectClass || sym.owner == AnyClass) Nil
```

Comment:
`def ##: Int`, as opposed to `def ##(): Int` has type `Int`, rather than `(): Int`, so with the change to `##` this was suddenly printing all the non-Any/Object members of Int!  So I opted for filtering out but I could also just update the check file without changes - lemme know.